<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Setting</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon_2.jpg</normaloff>icon_2.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="accessibleName">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(240, 220, 255);</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="grading_tab">
       <attribute name="title">
        <string>Grading </string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
              <property name="text">
               <string>نام فایل های تمرین (به فرمت csv)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="gs_file_names_lineEdit">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 255);</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QSplitter" name="splitter_6">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <widget class="QSplitter" name="gs_beauty_splitter">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QSlider" name="gs_beauty_hSlider">
              <property name="minimumSize">
               <size>
                <width>300</width>
                <height>35</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>550</width>
                <height>35</height>
               </size>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
             <widget class="QLabel" name="gs_beauty_label2">
              <property name="minimumSize">
               <size>
                <width>35</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>35</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
             <widget class="QLabel" name="gs_beauty_label1">
              <property name="minimumSize">
               <size>
                <width>140</width>
                <height>35</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>180</width>
                <height>35</height>
               </size>
              </property>
              <property name="text">
               <string>نمره زیبایی: </string>
              </property>
             </widget>
            </widget>
            <widget class="QSplitter" name="gs_report_splitter">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QSlider" name="gs_report_hSlider">
              <property name="minimumSize">
               <size>
                <width>300</width>
                <height>35</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>550</width>
                <height>35</height>
               </size>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
             <widget class="QLabel" name="gs_report_label2">
              <property name="minimumSize">
               <size>
                <width>35</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>35</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
             <widget class="QLabel" name="gs_report_label1">
              <property name="minimumSize">
               <size>
                <width>140</width>
                <height>35</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>180</width>
                <height>35</height>
               </size>
              </property>
              <property name="text">
               <string>نمره فرمت گزارش:</string>
              </property>
             </widget>
            </widget>
            <widget class="QSplitter" name="gs_splitter">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QSlider" name="gs_hSlider">
              <property name="minimumSize">
               <size>
                <width>300</width>
                <height>35</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>550</width>
                <height>35</height>
               </size>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
             <widget class="QLabel" name="gs_label2">
              <property name="minimumSize">
               <size>
                <width>35</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>35</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
             <widget class="QLabel" name="gs_label1">
              <property name="minimumSize">
               <size>
                <width>140</width>
                <height>35</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>180</width>
                <height>35</height>
               </size>
              </property>
              <property name="text">
               <string>ذخیره</string>
              </property>
             </widget>
            </widget>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
              <property name="text">
               <string>شماره تست ها با خروجی تصویر</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="gs_img_output_lineEdit">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 255);</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>تغیر امتیاز تست ها</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLineEdit" name="gs_test_1_lineEdit">
                  <property name="minimumSize">
                   <size>
                    <width>50</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>50</width>
                    <height>30</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_11">
                  <property name="text">
                   <string>نمره تست:  </string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>80</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QSpinBox" name="gs_test_1_spinBox"/>
                </item>
                <item>
                 <widget class="QLabel" name="label_9">
                  <property name="text">
                   <string>شماره تست:  </string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLineEdit" name="gs_test_2_lineEdit">
                  <property name="minimumSize">
                   <size>
                    <width>50</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>50</width>
                    <height>30</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_12">
                  <property name="text">
                   <string>نمره تست:  </string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>80</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QSpinBox" name="gs_test_2_spinBox"/>
                </item>
                <item>
                 <widget class="QLabel" name="label_10">
                  <property name="text">
                   <string>شماره تست:  </string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="output_tab">
       <attribute name="title">
        <string>Output</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QSplitter" name="splitter">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QCheckBox" name="os_excel_checkBox">
             <property name="text">
              <string>Excel</string>
             </property>
            </widget>
            <widget class="QCheckBox" name="os_pdf_checkBox">
             <property name="text">
              <string>PDF</string>
             </property>
            </widget>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>خروجی فایل:</string>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QRadioButton" name="os_pyDocker_radioButton">
                <property name="text">
                 <string>Docker</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="os_pyDefault_radioButton">
                <property name="text">
                 <string>System Default</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>نحوه ی اجرای کد پایتون:</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="os_grade_checkBox">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="text">
             <string>خروجی نمره خام</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="os_mos_checkBox">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="text">
             <string>MOS خروجی</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="os_mos_grade_checkBox">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="text">
             <string>MOS خروجی نمره با اعمال تاثیر</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSplitter" name="splitter_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QSlider" name="os_cheat_hSlider">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>550</width>
               <height>35</height>
              </size>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
            <widget class="QLabel" name="os_cheat_label">
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
            <widget class="QLabel" name="label_6">
             <property name="minimumSize">
              <size>
               <width>130</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>حد درصد شباهت:</string>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="advance_tab">
       <attribute name="title">
        <string>Advance</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>کلید واژه ی build موفقیت آمیز</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="adv_bulid_lineEdit">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Successfully built</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>32</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>کلید واژه ی صحت تست در cpp </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="adv_cppTest_lineEdit">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>&lt;&lt;&lt;success&gt;&gt;&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>32</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>کلید واژه ی عدم صحت تست در python</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="adv_pyTest_lineEdit">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Failure</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>32</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>کلید واژه ی کسر نمره در هر تست</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="adv_minus_lineEdit">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>minus</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="cancel_Button">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ok_Button">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
